lib/buildlog/mod.rs

```diff
@@ -34,6 +42,59 @@ impl BuildLog {
 		}
 	}
 
+	pub fn estimated_total_task_time(
```

Comment:
This should probably not be part of lib/buildlog/. This module just does reading/parsing and writing of the build log files, and should not care about how things are estimated. Instead, it could provide an `.entries()` or something that simply iterates over the entries, so this function can be moved to `status.rs`.

Reply:
Same for `average_historic_task_time`.